src/components/ResultsList.vue

```diff
@@ -1,23 +1,20 @@
 <template>
   <section class='results-list'>
-    <h1>Result List</h1>
-    <section :key='i' v-for='(result, i) in searchResults'>
+    <h4 v-if="!$store.state.searchResults.length">
+      <h5>No results found</h5>
+      <FindPark />
```

Comment:
This makes for an intuitive user experience so they don't have to navigate back to the homepage. I wouldn't have thought about this solution. Thanks for this!